setup.py

```diff
@@ -20,6 +20,7 @@
         "robotframework-requests",
         "robotframework-seleniumlibrary",
         "robotframework-jupyterlibrary>=0.3.1",
+        "robotframework-openshift",
```

Comment:
It should include exact library version so we control which version ods-ci is using